_data/site.yaml

```diff
@@ -37,6 +37,8 @@ dap:
 secondary_navigation:
   - name: Contact us
     url: "/contact/"
+  - name: Policies
```

Comment:
We don't need this up here, just in the footer is fine. For the jumplinks in the sidebar I was referring to see the cold case policy page: https://www.coldcaserecords.gov/policies/#accessibility-statement